### PR TITLE
Add golden tests

### PR DESCRIPTION
**Pull request by anothertobi**
## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

**Comment by anothertobi** [comment]
Didn't update the `Makefile` on purpose, since this is managed by ModuleSync.

File: .sync.yml [hunk 1]
```diff
@@ -1,2 +1,5 @@
 :global:
   componentName: CSI driver SMB
+  githubUrl: https://github.com/projectsyn/component-csi-driver-smb
```
Comment (simu) [inline .sync.yml:3]: This is not needed at all

```suggestion
```

File: .sync.yml [hunk 2]
```diff
@@ -1,2 +1,5 @@
...
+  feature_goUnitTests: false
```
Comment (simu) [inline .sync.yml:4]: This is the default, so it doesn't need to be specified.
```suggestion
```

**Review by simu — CHANGES_REQUESTED** [review]

**Review by simu — APPROVED** [review]
eclint will not complain about errors in the golden test outputs once you get the updated config from modulesync IIRC

**Comment by anothertobi** [comment]
Related ModuleSync PR: https://github.com/projectsyn/component-csi-driver-smb/pull/25

**Pull request merged**